src/session.rs

```diff
@@ -131,6 +135,43 @@ pub fn create_batch(queries: Vec<&PreparedStatementWrapper>) -> BatchWrapper {
     BatchWrapper { inner: batch }
 }
 
+fn apply_options(
+    mut query: PreparedStatement,
+    options: &QueryOptionsWrapper,
+) -> napi::Result<PreparedStatement> {
+    if let Some(o) = options.consistency {
+        query.set_consistency(
+            Consistency::try_from(o)
+                .map_err(|_| js_error(format!("Unknown consistency value: {o}")))?,
+        );
+    }
```

Comment:
Let's add support for `serial_consistency`, too, as it's going to be fully analogous.